templates/types/streaming/nextjs/app/components/ui/chat/use-config.ts

```diff
@@ -0,0 +1,27 @@
+"use client";
+
+import { useEffect, useMemo, useState } from "react";
+
+export interface ChatConfig {
```

Comment:
How about adding the vale of NEXT_PUBLIC_CHAT_API to this config object?

Reply:
Because this the model for the data from BE so i think the BE doesn't need to know what is the NEXT_PUBLIC_CHAT_API

Reply:
This is client code right? I mean we should centralize the usage of NEXT_PUBLIC_CHAT_API at one point